README.md

```diff
@@ -50,27 +67,28 @@ deployer also serves as a CI agent.
     3.  Set `env` to `staging` or `production` depending on which set of secrets
         you want to use. If set to `production`, DNS will not be configured, so
         the next two steps can be skipped.
-    4.  Set `domain` to the domain under which the domain names for the machines
-        should be created.
-    5.  Set `dnsZone` to the AWS DNS zone in which these domains can be
+    4.  Set `domain` to the full domain name under which the records for
+        the names of the machines should be created.
+    5.  Set `dnsZone` to the name of an AWS DNS zone in which these records will be
         configured if it differs from `disciplina.io.`.
 
-8.  Run `scripts/setdeps.sh 'https://github.com/DisciplinaOU/'
+8.  Run `./scripts/setdeps.sh 'https://github.com/DisciplinaOU/'
```

Comment:
I don't see the point of adding `./`.

Reply:
Yeah, I guess you are right. (I just like the dot 🤷‍♂️.)